backend/core/src/main/java/org/sonarsource/sonarlint/core/hotspot/HotspotService.java

```diff
@@ -50,7 +50,7 @@
 public class HotspotService {
 
   private static final SonarLintLogger LOG = SonarLintLogger.get();
-  private static final String NO_BINDING_REASON = "The project is not bound, please bind it to SonarQube 9.9+ or SonarCloud";
+  private static final String NO_BINDING_REASON = "The project is not bound, please bind it to SonarQube (Server, Cloud)";
```

Comment:
Shouldn't we mention here the dependency on SonarQube Server 9.9+ at least?

Reply:
We only support SQ 9.9 as minimal version now, so this is not relevant anymore